README.md

```diff
@@ -39,34 +45,32 @@ use VanOns\Laraberg\Traits\RendersContent;
 class Post extends Model
 {
     use HasFactory, RendersContent;
-    
+
     protected $contentColumn = 'content';
-       
+
     ...
 }
 ```
 
-
-
 Call the render method on your model in a template.
 
-
 ```php
 {!! $model->render() !!}
 ```
 
-### Options
+#### Options
```

Comment:
Ik vraag me af of het `Usage` deel vanaf hier tot `Documentation` hier ook moet, of dat het beter naar de uitgebreide documentatie kan? Ik denk dat het straks, als de docs site live is, tof zou zijn als we mensen daar naartoe kunnen laten komen (bijvoorbeeld door uitgebreide documentatie daar te plaatsen). Ook hier de vraag: wat denk jij?

Reply:
I think for detailed documentation, it might be handier to host something like [mkdocs](https://www.mkdocs.org/), and put detailed documentation it there. It is a lot easier to navigate than lengthy markdown files.